src/types/venues.ts

```diff
@@ -83,6 +83,7 @@ export interface Venue_v2_AdvancedConfig {
   radioStations?: string | string[]; // single string on form, array in DB
   requiresDateOfBirth?: boolean;
   roomVisibility?: RoomVisibility;
+  type?: boolean;
```

Comment:
`type` is defined on `Room`, not `Venue`.

```suggestion
```

Reply:
🤦‍♂️